tests/fixtures/brocfile-tests/app-test-import/node_modules/ember-test-addon/index.js

```diff
@@ -12,9 +12,9 @@ class AddCustomFile extends Plugin {
   }
 
   build() {
-    let outPath = path.join(this.path);
-    this.output.mkdirSync(path.dirname(outPath));
-    this.output.writeFileSync(outPath, '// File for test tree imported and added via postprocessTree()');
+    let outPath = path.join(this.outputPath, this.path);
```

Comment:
I know this is the "old" way of doing this, but the "new" way fails under pnpm. I assume because during the test, `fs-extra` isn't found / installed.